hotham/src/asset_importer/mod.rs

```diff
@@ -134,7 +155,17 @@ fn load_models_from_gltf_data(import_context: &mut ImportContext) -> Result<()>
     // We need *some* entity to stash the AnimationController onto.
     // For now, just use the first root entity.
     let mut animation_controller_entity = None;
-    for node in document.scenes().next().unwrap().nodes() {
+
+    // NOTE: We don't currently support multiple scenes, so we just take the first one.
+    let scene = document.scenes().next().unwrap();
+
+    // Iterate through each of the root nodes in the scene and load it in.
+    for node in scene.nodes() {
+        // Don't add wall collider geometry as nodes.
+        if node.name().unwrap_or_default().contains(WALL_COLLIDER_TAG) {
```

Comment:
You might want to use `ends_with` instead of `contains` here and elsewhere.

Reply:
That's a good point! There's only one case where this doesn't work (when we're skipping importing a mesh that could be either type of collider), but it works everywhere else!